Project.toml

```diff
@@ -0,0 +1,23 @@
+name = "DiffEqBayes"
+uuid = "ebbdde9d-f333-5424-9be2-dbf1e9acfb5e"
+authors = ["Vaibhavdixit02 <vaibhavyashdixit@gmail.com>"]
+version = "1.2.0"
+
+[deps]
+ApproxBayes = "f5f396d3-230c-5e07-80e6-9fadf06146cc"
+DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+DynamicHMC = "bbc10e6e-7c05-544b-b16e-64fede858acb"
```

Comment:
Missing the things like LinearAlgebra and stuff like that. Look in the repo for what you need. Things will throw warnings without it.